arch/x86/kvm/pkvm/pgtable.h

```diff
@@ -167,4 +167,28 @@ int pkvm_pgtable_init(struct pkvm_pgtable *pgt,
 int pkvm_pgtable_walk(struct pkvm_pgtable *pgt, unsigned long vaddr,
 		      unsigned long size, struct pkvm_pgtable_walker *walker);
 
+/*
+ * pkvm_pgtable_map() - Install a vaddr to phys mapping in a pkvm pgtable.
+ *
+ * @pgt:	The page table to install mapping.
+ * @vaddr:	The virtual address of the installed mapping.
+ * @phys:	The physical address to map.
+ * @size:	The memory size to map.
+ * @prot:	The property bits to create the mapping.
+ *
+ * The vaddr/phys will be aligned down with PAGE_SIZE, and the size will
+ * be aligned with the PAGE_SIZE.
+ *
+ * The pkvm_pgtable_map can install vaddr to paddr mappings with the given
```

Comment:
s/can install/installs/, s/paddr/phys/ ?

Reply:
As we shall align the end address rather than the size, the comment of map/unmap API will also need to be updated accordingly, for line 180 as well.

I will change paddr to @phys.